openmls/src/extensions/codec.rs

```diff
@@ -115,6 +111,9 @@ impl Deserialize for Extension {
             ExtensionType::ExternalSenders => Extension::ExternalSenders(
                 ExternalSendersExtension::tls_deserialize(&mut extension_data)?,
             ),
+            ExtensionType::Unknown(unknown) => {
+                Extension::Unknown(unknown, UnknownExtension(extension_data.to_vec()))
+            }
         })
     }
 }
```

Comment:
Let's add at least one short test here to make sure (de)serializing of unknown extensions works.